spec/migrations/20171106143927_downcase_resource_group_ems_ref_for_azure.rb

```diff
@@ -0,0 +1,31 @@
+require_migration
+
+describe DowncaseResourceGroupEmsRefForAzure do
+  let(:resource_group_stub) { migration_stub :ResourceGroup }
+
+  let(:uppercase_stub) do
+    resource_group_stub.create!(:name => 'UPPER', :ems_ref => '/subscriptions/xyz/resourceGroups/UPPER')
+  end
+
+  let(:mixedcase_stub) do
+    resource_group_stub.create!(:name => 'MiXeD', :ems_ref => '/subscriptions/xyz/resourceGroups/MiXeD')
+  end
+
+  let(:lowercase_stub) do
+    resource_group_stub.create!(:name => 'lower', :ems_ref => '/subscriptions/xyz/resourceGroups/lower')
+  end
+
+  migration_context :up do
+    it 'Downcases the resource group ems_ref column' do
+      migrate
+      expect(uppercase_stub.ems_ref).to eql('/subscriptions/xyz/resourcegroups/upper')
```

Comment:
I'm curious how this works since it creates the record *after* the migration occurs (because you are using `let` instead of `let!`

Reply:
Oh I know how it "works"...it's because it's not running at all...the filename needs to be changed to have _spec.rb at the end